tests/unittests/graphOptzTest.cpp

```diff
@@ -872,3 +872,50 @@ TEST_F(GraphOptz, concatElim) {
   // Check that the concat node is gone.
   EXPECT_EQ(countNodeKind(F_, Kinded::Kind::ConcatNodeKind), 0);
 }
+
+// Check the transformation Concat(Reshape(x) * N) -> Reshape(Concat(x * N)).
+TEST_F(GraphOptz, concatReshapes) {
+  const size_t shape[] = {10, 20};
+  const size_t reshape[] = {10, 1, 20};
+  Type t(ElemKind::FloatTy, shape);
+  llvm::SmallVector<NodeValue, 10> inputs;
+  for (size_t i = 0; i < 10; i++) {
+    // Create 10 reshape nodes whose input is variable of shape {10,20}.
+    // The reshape nodes will transform their input from shape of {10,20}
+    // to {10, 1, 20}.
+    Node *var = F_->getParent()->createVariable(ElemKind::FloatTy, shape,
+                                                "input" + std::to_string(i));
+    auto *RN = F_->createReshape("reshape" + std::to_string(i), var, reshape);
+    inputs.push_back(RN);
+  }
+  // Create a Concat node that concatenates the 10 reshape nodes.
+  auto *ConcatN = F_->createConcat("concat", inputs, 1);
```

Comment:
nit: concatNode (or concatN, prefer the first one) according to the naming convention.

Make sure you update all comments accordingly. (CN should exist, etc).

Reply:
done